jobs/workflow_test_release.groovy

```diff
@@ -66,16 +66,21 @@ job(name) {
     colorizeOutput 'xterm'
     credentialsBinding {
       string("GCLOUD_CREDENTIALS", "246d6550-569b-4925-8cda-e11a4f0d6803")
+      string("AUTH_TOKEN", "a62d7fe9-5b74-47e3-9aa5-2458ba32da52")
```

Comment:
what's `AUTH_TOKEN`?  For authorizing the docker actions?  Any way we can name it a bit more descriptively?